### ISSUE TITLE
blacklisting facilities

### ISSUE THREAD

**Issue by motorBMW** [alexpevzner/sane-airscan#120]
I remember you were asking for blacklisting facilities. I was thinking some time about it and can probably add something like this:
```
[blacklist]
model = "Brother*"
name = "*LaserJet*"
ip = 192.168.0.1/24
```

_Originally posted by @alexpevzner in https://github.com/alexpevzner/sane-airscan/issues/97#issuecomment-725979875_

**Comment by motorBMW** [comment]
Hi, Alexander. I created the issue to highlight the need to create a blacklist. I would be very grateful for your help!

**Comment by alexpevzner** [comment]
OK. Sorry for delay in answering. Probably, I'll add it in 0.99.25

**Comment by motorBMW** [comment]
Hi, Alexander!

I really need your help! Please create a blacklist!

**Comment by alexpevzner** [comment]
You may test it for now, but please note, that it is not well-tested on my side yet :-)

The syntax is as proposed above. model/name supports glob-style wildcards. Netmask is optional, if omitted, exact address match is assumed.

**Comment by motorBMW** [comment]
The filter works fine, the necessary devices are excluded from the list of available devices. Now I look forward to your assembled package to distribute to the devices. Thank you very much for your work, Alexander!

**Comment by alexpevzner** [comment]
Fixed in 0.99.25, sorry for delay with release
